src/containers/WalletAdd/WalletAdd.tsx

```diff
@@ -72,7 +72,7 @@ class WalletAdd extends React.Component<Props, State> {
     let identity
     const phrase = useMyPhrase ? myPhrase : randomPhrase
     try {
-      identity = await Identity.buildFromMnemonic(phrase)
+      identity = Identity.buildFromMnemonic(phrase)
```

Comment:
`addIdentity` still async?

Reply:
Yes, we are fetching the `DID` and waiting for the promise to resolve.